contents/forms/dropzones/with-image-preview/index.tsx

```diff
@@ -30,47 +25,28 @@ export const ImagePreviewDropzone: FC = () => {
   }
 
   return (
-    <VStack>
+    <VStack maxW="5xl">
       <Dropzone
         multiple
         accept={{
           "image/*": [],
         }}
         onDropAccepted={handleAcceptedFile}
       >
-        <Text>Drag and Drop Image File</Text>
+        <Text>Drag and Drop Image Files</Text>
       </Dropzone>
 
-      <Carousel
-        visibility={imageURL.length > 0 ? "visible" : "hidden"}
-        align="center"
-        controlProps={{
-          background: "blackAlpha.500",
-        }}
-      >
+      <Wrap as={Center} gap="md">
         {imageURL.map((url, index) => (
-          <CarouselSlide key={index} as={Center} position="relative">
-            <Image
-              src={url}
-              w="full"
-              h="full"
-              objectFit="contain"
-              alt="image"
-            />
-          </CarouselSlide>
+          <Image
+            key={index}
+            src={url}
+            alt={`image-${index}`}
+            width={200}
+            height={200}
+          />
```

Comment:
You’re specifying the width and height directly with fixed values, so the photo may be displayed incorrectly depending on the aspect ratio. It might be better to set it to full to maintain the original aspect ratio

Reply:
What about `object-fit`?

Reply:
Sorry, I think that's better

Reply:
Added `objectFit`.